ncvoter/local_settings.example.py

```diff
@@ -0,0 +1,11 @@
+from .settings import *
+
+DATABASES['default'].update({
+    'PORT': 5455,
+    'NAME': 'ncvotes',
+    'USER': '',
+    'PASSWORD': '',
+})
+
+NCVOTER_DOWNLOAD_PATH = "/Volumes/Untitled/downloads/ncvoter"
+NCVHIS_DOWNLOAD_PATH = "/Volumes/Untitled/downloads/ncvhis"
```

Comment:
We use 'Untitled' here and in the docs, but 'CalvinNCVdb' in the shell script. Should we stick to one name throughout, to make it clearer to the developer?